resources/config-defaults.edn

```diff
@@ -472,7 +472,15 @@
  :plugins nil ; by default no plugins
  :extension-points nil ; by default no plugins
  :enable-voting false ; voting (experimental)
+ 
+ ;; If a `:malware-scanner-path` is defined, it is assumed to be an executable that:
+ ;; can take a binary file from `STDIN` and return a status code of zero if and only if the file passes the scan.
+ :malware-scanner-path nil
 
+ ;; If `:malware-scanner-path` is defined, enabling `:enable-malware-scanner-logging` will make REMS log whatever the
+ ;; malware executable printed to `STDERR` before exiting.
+ :enable-malware-scanner-logging false
+ 
```

Comment:
If there are going to be many options, one alternative is to group the config like so:
```clj
:malware-scanner {:path "..."
                  :logging false}
```

Reply:
fixed in [9eb5038](https://github.com/CSCfi/rems/pull/3202/commits/9eb503853f379b6019547650b137b5f7900d06c8)